_locales/en/messages.json

```diff
@@ -1750,7 +1750,7 @@
       "Shown in notifications and in the left pane when a message is a disappearing photo."
   },
   "message--getDescription--disappearing-video": {
-    "message": "Disappering Video",
+    "message": "Disappearing Video",
     "description":
       "Shown in notifications and in the left pane when a message is a disappearing photo."
```

Comment:
Can you change "is a disappearing photo" => "is a disappearing video" as well? Was initially rported by @mueller-ma (https://github.com/signalapp/Signal-Desktop/commit/5c00b89600df90c0e621eae0da45fe2627965b87#r35456719)